pkg/verification/service/verification_service.go

```diff
@@ -164,7 +164,7 @@ func (s *ServiceImpl) InitVerification(ctx *gin.Context, userID string, e164Phon
 	}
 
 	doUpdate := func() error {
-		signup, err := s.Services().SignupService().GetUserSignup(userID)
+		signup, err := s.Services().SignupService().GetUserSignup(userID, "")
```

Comment:
why is the username not passed here? 
```suggestion
		signup, err := s.Services().SignupService().GetUserSignup(userID, username)
```

Reply:
Good catch, fixed this.